zap/src/main/java/org/parosproxy/paros/core/scanner/ScannerParam.java

```diff
@@ -171,6 +175,8 @@
     private int targetParamsInjectable = TARGET_INJECTABLE_DEFAULT;
     private int targetParamsEnabledRPC = TARGET_ENABLED_RPC_DEFAULT;
 
+    private String whitelistingValueExpression = null;
```

Comment:
This should be a list of expressions, and better if they can be enabled/disabled as it's easier to temporarily include/exclude them.

Reply:
Should also be renamed